src/libpriv/rpmostree-core.c

```diff
@@ -2244,6 +2244,14 @@ apply_rpmfi_overrides (int            tmp_metadata_dfd,
           g_str_equal (group, "root"))
         continue;
 
+      /* In theory, RPMs could contain block devices or FIFOs; we would normally
+       * have rejected that at the import time, but let's also be sure here.
```

Comment:
Just to clarify, what I meant was that we don't actually reject this at import time. We use `ignore_unsupported_content`, which tells ostree to skip over them. So the conditional here is just to make sure we only apply overrides on files we actually did import.